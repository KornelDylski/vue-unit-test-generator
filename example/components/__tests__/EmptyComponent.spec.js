import { shallowMount } from '@vue/test-utils';
import EmptyComponent from '../EmptyComponent.vue';

/**
 * EmptyComponent.vue unit tests
 */
describe('EmptyComponent', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(EmptyComponent);
  });

});
