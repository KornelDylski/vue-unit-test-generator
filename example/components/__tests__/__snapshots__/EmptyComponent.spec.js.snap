// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyComponent should match snapshot 1`] = `
<p>
  
  Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias
  repudiandae quasi, eligendi in tempora veritatis sapiente alias modi, saepe
  possimus, odio ex labore sint facilis voluptatem officia ea iusto maxime.

</p>
`;
